index.html

```diff
@@ -106,13 +109,12 @@ <h2>Publish simplified infrastructure abstractions for your applications</h2>
     </div>
     <div class="grid">
       <div class="col-6_md-12">
-        <h2>Deploy applications using a team centric approach with OAM</h2>
+        <h2>Universal Cloud API</h2>
         <p>
-          Crossplane implements the Open Application Model to help unify application and infrastructure management around team-centric processes. With Crossplane and OAM, 
-          application and infrastructure config can co-exist and be deployed using the same tooling.
-        </p>
-        <p>
-          <a href="https://oam.dev">Learn More About OAM</a>
+          Crossplane provides a consistent API across a diverse set of vendors, resources, and
+          abstractions. The Crossplane Resource Model (XRM) extends the Kubernetes Resource Model
+          (KRM) in an opinionated way, resulting in a universal experience for managing resources,
+          no matter what provider or level of abstraction they are a part of.
```

Comment:
Suggestion: "no matter what provider or vendor built them"

The "level of abstraction" is hard for me to grok.